src/GafferScene/RenderManifest.cpp

```diff
@@ -283,32 +347,40 @@ void RenderManifest::loadFromImageMetadata( const IECore::CompoundData *metadata
 		throw IECore::Exception( std::string( "Could not find manifest file : " ) + sideCarManifestPath + " : " + e.what() );
 	}
 
-	if( sideCarManifestPath == m_loadedFromFilePath && currentModTime == m_loadedFromFilePathModTime )
+
+	CacheKey cacheKey( nullptr, sideCarManifestPath, currentModTime );
```

Comment:
It does feel like it would be simpler to have two caches, one for JSON strings and one for EXR files. The CacheKey has various states that don't make sense (like having both a JSON string and a filename). If there were two caches I think one would use ConstStringPtr as the key, with simpler custom equality and hash functions, and the other could just use a plain `pair<string, time>`. Not particularly important though...